backend/cms/templates/_base.html

```diff
@@ -30,15 +30,19 @@
 	        {% endif %}
         </div>
         <div id="instance-selector-list" class="absolute shadow rounded-b">
-	        {% for other_region in sites %}
-	            <a href="{% url 'dashboard' site_slug=other_region.slug %}" class="block px-4 py-3 text-grey-darkest hover:bg-grey">
-	                {{ other_region.name }}
-	            </a>
+	        {% for other_region in user.profile.regions.all %}
+		        {% if not other_region == site %}
```

Comment:
Why is the current site excluded here? Did I miss a convention we agreed on or does this have technical reasons?

Reply:
This is only the region selector in the frontend. The user is already operating on the current site (which is displayed in the div above), thus we don't have to provide a link to go to the current site.

Reply:
Ok, I just pulled the corresponding branch and tested it myself. Now I get the reason, I just wasn't aware where this div is displayed and thought of a (complete) list in the body of the webpage. Thanks for the clarification!